persist/fs/commitlog/commit_log.go

```diff
@@ -99,14 +99,16 @@ type commitLogWrite struct {
 	datapoint    ts.Datapoint
 	unit         xtime.Unit
 	annotation   ts.Annotation
+	ctx          context.Context
```

Comment:
I've put it here since it sort of belongs with `completionFn`, but if there's a new lint rule, then yeah makes sense to move it.